examples/Remarketing/AddMerchantCenterDynamicRemarketingCampaign.cs

```diff
@@ -0,0 +1,400 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Linq;
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.Util;
+using Google.Ads.GoogleAds.V3.Common;
+using Google.Ads.GoogleAds.V3.Errors;
+using Google.Ads.GoogleAds.V3.Resources;
+using Google.Ads.GoogleAds.V3.Services;
+using Google.Protobuf;
+using static Google.Ads.GoogleAds.V3.Enums.AdvertisingChannelTypeEnum.Types;
+using static Google.Ads.GoogleAds.V3.Enums.CampaignStatusEnum.Types;
+using static Google.Ads.GoogleAds.V3.Enums.DisplayAdFormatSettingEnum.Types;
+using static Google.Ads.GoogleAds.V3.Enums.AdGroupStatusEnum.Types;
+using static Google.Ads.GoogleAds.V3.Enums.AssetTypeEnum.Types;
+
+namespace Google.Ads.GoogleAds.Examples.V3
+{
+    /// <summary>
+    /// Creates a shopping campaign associated with an existing merchant center account, along
+    /// with a related ad group and dynamic display ad, and targets a user list for remarketing
```

Comment:
I noticed the code does responsive display ad, and this says dynamic display ad. Can we stick with one terminology, so folks don't get confused? The terminology is so similar for so many ad types.

Reply:
Good point, updated the comments to replace "dynamic" with "responsive."